php/libraries/parsedown/README.md

```diff
@@ -0,0 +1,103 @@
+<!-- ![Parsedown](https://i.imgur.com/yE8afYV.png) -->
```

Comment:
I'm inclined to say delete this file too, but I can be convinced it should stay if anyone feels strongly about it